.github/workflows/labeler.yml

```diff
@@ -0,0 +1,26 @@
+name: Label PR
+on:
+  pull_request:
+    types: [opened, edited]
+
+jobs:
+  label-pr:
+    if: github.event_name == 'pull_request' && github.event.pull_request.head.repo.fork == false
```

Comment:
We can't run this action on PRs created from forks since those don't have write access to the repo.